buildSrc/build.gradle.kts

```diff
@@ -64,18 +64,23 @@ apply<GradleVersionsFilterPlugin>()
 apply<VersionCatalogUpdatePlugin>()
 apply<KotlinPluginWrapper>()
 
-val javaMajorVersion = JavaVersion.VERSION_11
+// Let's use a low version so that buildSrc/ itself builds & executes properly
+// on every machine even when running for the first time.
+// In the top-level Gradle config, there is a Gradle toolchain,
+// which makes sure that the project itself builds under the correct (high) Java version,
+// even if it was previously missing from the machine.
+val buildSrcJavaVersion = JavaVersion.VERSION_1_8
```

Comment:
So now the project will build OOTB even if the machine has only Java 8 at the start (which is kind of absolute minimum anyway ;p).

Reply:
TIL that Java 8 [will live longer than Java 11](https://www.oracle.com/java/technologies/java-se-support-roadmap.html):

![image](https://user-images.githubusercontent.com/3383210/193429040-9c7f6906-2eb9-44e1-b87d-e1c7f46c1734.png)

Reply:
why cannot we have toString() here?